src/cli.ts

```diff
@@ -5,7 +5,16 @@ import mri from 'mri'
 import { resolve } from 'pathe'
 import { buildModule } from './build'
 
-async function main () {
+// TODO: use citty
+function main () {
+  const args = mri(process.argv.slice(2))
+  if ('prepare' in args) {
```

Comment:
Why not making it a dedicated sub command? (Also that this is preparing for module itself not essentially a playground)